charts/kafka/templates/statefulset.yaml

```diff
@@ -12,65 +12,64 @@
 #
 # Copied from: https://github.com/helm/charts/blob/6c85be7b88748171afd17affe8b1b57c66bf66a2/incubator/kafka/templates/statefulset.yaml
```

Comment:
update the header. Need to add the crown copyright and change the 'Copied from: ...' to 'Originally based on ...'